middleware/app.py

```diff
@@ -67,7 +67,7 @@ def ubi_events():
 
     if request.method == "OPTIONS":
         response = flask.jsonify(status=200, mimetype="application/json")
-        response.headers.add("Access-Control-Allow-Origin", "http://{OPENSEARCH_HOST}:4000")
+        response.headers.add("Access-Control-Allow-Origin", "*")
```

Comment:
Did this work ok? I'm not well-versed enough in CORS but from the docs it looks like it should! (But that's CORS, right?)